extensions/sql-database-projects/src/controllers/mainController.ts

```diff
@@ -57,7 +57,12 @@ export default class MainController implements vscode.Disposable {
 		vscode.commands.registerCommand('sqlDatabaseProjects.publish', async (node: WorkspaceTreeItem) => { this.projectsController.publishProject(node); });
 		vscode.commands.registerCommand('sqlDatabaseProjects.deployLocal', async (node: WorkspaceTreeItem) => { return this.projectsController.deployProject(node); });
 		vscode.commands.registerCommand('sqlDatabaseProjects.schemaCompare', async (node: WorkspaceTreeItem) => { return this.projectsController.schemaCompare(node); });
-		vscode.commands.registerCommand('sqlDatabaseProjects.createProjectFromDatabase', async (context: azdataType.IConnectionProfile | vscodeMssql.ITreeNodeInfo | undefined) => { return this.projectsController.createProjectFromDatabase(context); });
+		vscode.commands.registerCommand('sqlDatabaseProjects.schemaCompareGetTargetScripts', async (projectFilePath: string): Promise<string[]> => { return await this.projectsController.schemaCompareGetTargetScripts(projectFilePath); });
+		vscode.commands.registerCommand('sqlDatabaseProjects.schemaCompareGetDsp', async (projectFilePath: string): Promise<string> => { return await this.projectsController.schemaCompareGetDsp(projectFilePath); });
```

Comment:
I think it makes more sense for these to be exposed as apis in sqldbproj.d.ts so that the schema compare extension can call those instead of running these commands.